benchmarks/benches/lib.rs

```diff
@@ -650,6 +651,30 @@ fn insert_range_bitmap(c: &mut Criterion) {
     }
 }
 
+fn insert_range_treemap(c: &mut Criterion) {
+    for &size in &[10, 100, 1_000, 5_000, 10_000, 20_000] {
```

Comment:
Could we add bigger values here?

Reply:
I've added 2 * (2^32 -1) now to ensure that we're crossing a bitmap boundary. Is there any point in adding an additional benchmark that compares the new `insert_range` to `(0..size).collect<RoaringTreemap>()`? Gut feeling says it's gonna be awfully slow with the current implementation.